### ISSUE TITLE
Incompatibility with webtrees 2.1.8 when adding a new person

### ISSUE THREAD

**Issue by hartenthaler** [vesta-webtrees-2-custom-modules/vesta_classic_laf#102]
When adding a new wife to an existing man: 
```
Undefined variable $cancel_url …/modules_v4/vesta_classic_look_and_feel/resources/views/edit/new-individual.phtml:24
#0 …/modules_v4/vesta_classic_look_and_feel/resources/views/edit/new-individual.phtml(24): Fisharebest\Webtrees\Webtrees::Fisharebest\Webtrees\{closure}()
#1 …/app/View.php(183): include('...')
#2 …/app/View.php(278): Fisharebest\Webtrees\View->render()
#3 …/app/Helpers/functions.php(145): Fisharebest\Webtrees\View::make()
#4 …/app/Http/ViewResponseTrait.php(50): view()
#5 …/app/Http/RequestHandlers/AddSpouseToIndividualPage.php(100): Fisharebest\Webtrees\Http\RequestHandlers\AddSpouseToIndividualPage->viewResponse()
```

**Comment by ric2016** [comment]
Fixed in Vesta release 2.1.8.0.1.

**Comment by hartenthaler** [comment]
Now it is ok Thank you!
